Packs/PhishLabs/Integrations/PhishLabsIOC_EIR/PhishLabsIOC_EIR.py

```diff
@@ -358,9 +359,9 @@ def fetch_incidents_command(
                                             sort='created_at',
                                             direction='asc')
     # Gather incidents by demisto format
-    new_last_run: Optional[str] = None
     incidents_report = []
     if incidents_raw:
+        demisto.debug(f'Got {len(incidents_raw)} incidents from the API.')
```

Comment:
not super critical but:
```suggestion
    incidents_report = []
    demisto.debug(f'Got {len(incidents_raw or [])} incidents from the API.')
    if incidents_raw:
```